cypress/support/step_definitions/dictionary/addConceptInDictionaryPage/addconceptInDictioanryPage.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import {
+  Given,
+  Then,
+  When
+} from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser, getConceptId } from "../../../utils";
+
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+});
+
+When("the user clicks the Add concepts button", () => {
+  cy.findByTitle("Add concepts").click();
+});
+
+Then("the user should be on the Import existing concept page", () => {
+  cy.visit("/orgs/CIEL/sources/CIEL/concepts/?addToDictionary=/users/openmrs/collections/")
```

Comment:
This should be something like `cy.url().should("contain", "<url>")`. `cy.url()` gets the current URL. `cy.visit()` redirects the user to the provided url.

Reply:
Also, instead of hard-coding `openmrs` as the user here, we should use `${getUser()}`. Note that to support this the string should be surrounded by back-ticks `` ` ``. (The difference between normal strings defined with `'` or `"` and strings defined with `` ` `` is that the strings defined with `` ` `` allow this kind of inline string interpolation.